Cubical/Structures/Ideal.agda

```diff
@@ -0,0 +1,78 @@
+{-# OPTIONS --cubical --safe #-}
+
+module Cubical.Structures.Ideal where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Equiv
+open import Cubical.Foundations.HLevels
+open import Cubical.Foundations.Logic using ([_])
+open import Cubical.Structures.Ring
+
+private
+  variable
+    ℓ ℓ′ : Level
+
+infix 5 _∈_
+_∈_ : {X : Type ℓ } → (x : X) → (P : X → hProp ℓ′) → Type _
```

Comment:
@mzeuner : Didn't you add this somewhere else in the library?

Reply:
Never mind, I found it: https://github.com/agda/cubical/blob/master/Cubical/Foundations/Logic.agda#L282

Reply:
Good, I'll use that then (and fix the universe level to be the same for the props and the type).